src/components/MessageHTMLBody.vue

```diff
@@ -4,6 +4,10 @@
 		<div v-if="hasBlockedContent" id="mail-message-has-blocked-content" style="color: #000000">
 			{{ t('mail', 'The images have been blocked to protect your privacy.') }}
 			<Actions default-icon="icon-toggle" :menu-title="t('mail', 'Show')">
+				<ActionButton icon="icon-toggle"
+					@click="displayIframe">
+					{{ t('mail', 'Show images temporarily from {sender}', {sender: message.from[0].email}) }}
```

Comment:
```suggestion
					{{ t('mail', 'Show images temporarily}) }}
```

No need to repeat who the message is from